src/router.js

```diff
@@ -333,6 +335,16 @@ export class Router {
     return undefined;
   }
 
+  _transformTitle(title: string) {
+    if (this.transformTitle) {
+      return this.transformTitle(title);
+    }
+    if (this.parent) {
+      return parent.transformTitle(title);
```

Comment:
I changed that in reaction to some comments. Instead of leaving a ```transformTitle``` property and a ```_transformTitle``` method trying to use this property, I removed the ```_transformTitle``` method and put its implementation as the default value of the ```transformTitle``` property. I feel like it's clearer, and it's lesser lines of code.
Do you see any downsides I missed?